test/scripts/measures/lib/merge-ecqm-ehr-links-spec.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const mergeEcqmEhrLinks = require('../../../../scripts/measures/lib/merge-ecqm-ehr-links');
+
+describe('merge-ecqm-ehr-links', () => {
+  it('should merge ecqmEhrLinks into base measures data', () => {
+    const measures = [
+      { eMeasureId: '0', measureSpecification: {} },
+      { eMeasureId: '1', measureSpecification: {} },
+      { eMeasureId: '2', measureSpecification: {} }
+    ];
+    const ecqmEhrLinks = [
+      { eMeasureId: '0', link: 'https://some.link/0' },
+      { eMeasureId: '1', link: 'https://some.link/1' }
+    ];
+    const expectedMeasures = [
+      { eMeasureId: '0', measureSpecification: { electronicHealthRecord: 'https://some.link/0' } },
+      { eMeasureId: '1', measureSpecification: { electronicHealthRecord: 'https://some.link/1' } },
+      { eMeasureId: '2', measureSpecification: {} }
+    ];
+
+    mergeEcqmEhrLinks(measures, ecqmEhrLinks);
+    expect(measures).to.deep.equal(expectedMeasures);
+  });
+
+  it('should not mutate base measures if measure(s) are not found', () => {
```

Comment:
I don't quite understand this test.  It says "if measures(s) are not found", but the only difference appears to be that there's not `measureSpecification` object.

Reply:
`mergeEcqmEhrLinks ` looks up measures based on `eMeasureId` instead of `measureId` (the id used in `measures` in this test case).

Reply:
Whoops!